README.md

```diff
@@ -27,3 +27,23 @@ Path | Method | Description
 [1]: http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
 [2]: https://maven.apache.org/download.cgi
 [3]: https://git-scm.com/downloads
+
+## Terraform ECS
+### What does this code do?
+The code present in this repository is used to define and deploy a dockerised container in AWS ECS.
+This is done by calling a [module](https://github.com/companieshouse/terraform-modules/tree/main/aws/ecs) from terraform-modules. Application specific attributes are injected and the service is then deployed using Terraform via the CICD platform 'Concourse'.
+Application specific attributes | Value                                | Description
+:---------|:-----------------------------------------------------------------------------|:-----------
+**ECS Cluster**        |order-service                                      | ECS cluster (stack) the service belongs to
+**Load balancer**      |{env}-apichgovuk <br> {env}-apichgovuk-private                                              | The load balancer that sits in front of the service
```

Comment:
Just need to remove this

Reply:
Well seen, makes totally sense. Thank you.